src/cli/index.ts

```diff
@@ -28,6 +29,10 @@ export async function main(): Promise<void> {
       process.exit(ExitCode.Success)
     }
     else {
+      if (!options.all) {
```

Comment:
We should probably also add a flag like `--no-git-check` to opt-out

Reply:
`all = false` && `noGitCheck = true` ?

Reply:
Yeah, like that

Reply:
okk

Reply:
done